Decred Wallet/Features/Security/PasswordSetupViewController.swift

```diff
@@ -9,19 +9,35 @@
 import UIKit
 
 class PasswordSetupViewController: SecurityBaseViewController, UITextFieldDelegate {
-    @IBOutlet weak var headerText: UILabel!
-    @IBOutlet weak var tfPassword: UITextField!
-    @IBOutlet weak var tfConfirmPassword: UITextField!
-    @IBOutlet weak var lbMatchIndicator: UILabel!
+    // Input fields
+    @IBOutlet weak var tfPassword: FloatingLabelTextInput!
+    @IBOutlet weak var tfConfirmPassword: FloatingLabelTextInput!
+    
     @IBOutlet weak var pbPasswordStrength: UIProgressView!
     
+    @IBOutlet weak var passwordCount: UILabel!
+    @IBOutlet weak var confirmCount: UILabel!
+    
+    // Buttons
```

Comment:
It's obvious from the variable names that these are buttons, comment is unnecessary.

Reply:
resolved in ab83980